cinema/views.py

```diff
@@ -1,15 +1,23 @@
 from datetime import datetime
 
 from django.db.models import F, Count
+
+from drf_spectacular.utils import OpenApiParameter, extend_schema
+
 from rest_framework import viewsets, mixins, status
-from rest_framework.authentication import TokenAuthentication
+
 from rest_framework.decorators import action
+
 from rest_framework.pagination import PageNumberPagination
+
 from rest_framework.permissions import IsAuthenticated, IsAdminUser
+
 from rest_framework.response import Response
-from rest_framework.viewsets import GenericViewSet, ReadOnlyModelViewSet
+
+from rest_framework.viewsets import GenericViewSet
 
 from cinema.models import Genre, Actor, CinemaHall, Movie, MovieSession, Order
+
 from cinema.permissions import IsAdminOrIfAuthenticatedReadOnly
 
```

Comment:
why do you need to separate each import with empty line?

Reply:
I thought it was not forbidden.